src/javascript/query-explorer/index.js

```diff
@@ -165,6 +175,14 @@ function handleDataChartError(err) {
       error: err.error
     }
   });
+
+  ga('send', {
+    hitType: 'event',
+    eventCategory: 'query',
+    eventAction: 'submit:error',
+    eventLabel: err.error.message,
```

Comment:
as per my other comment, you may want to include the http code here. E.g. (403) <error message>